superset-frontend/src/views/CRUD/data/database/state.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { useSelector, TypedUseSelectorHook } from 'react-redux';
+import { RootState } from './types';
+
+export const useTypedSelector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
essentially what you want to do here is just pull the RootState (state.common.conf) out of redux like this: https://github.com/apache/superset/blob/master/superset-frontend/src/dashboard/components/nativeFilters/state.ts#L25